LICENSE

```diff
@@ -1,16 +1,16 @@
-Limerick Permissive License (LPL v2)
-
-(c) 2022 Alexander E Genaud
-
-Permission is granted hereby,
-to copy, share, use, modify,
-   for purposes any,
-   for free or for money,
-provided these rhymes catch eye.
-
-This work "as is" I provide,
-no warranty express or implied,
-   for, no purpose fit,
-   'tis unmerchantable shit.
-Liability for damages denied.
-
+ Limerick Permissive License (LPL v2)
+ 
+ (c) 2022 Alexander E Genaud
+ 
+ Permission is granted hereby,
+ to copy, share, use, modify,
+    for purposes any,
+    for free or for money,
+ provided these rhymes catch eye.
+ 
+ This work "as is" I provide,
+ no warranty express or implied,
+    for, no purpose fit,
+    'tis unmerchantable shit.
+ Liability for damages denied.
+ 
```

Comment:
I've recently reversed the stanzas to parallel the ISC and MIT licenses. Does it make any difference? Is the order arbitrary? All licenses offer software for any purpose then later or before assert no fitness for any purpose. They perfectly contradict, whether in this order and in that way.

For a dozen years, I had the warranty above the permissions and conditions. I prefer that arrangement because the first line of the warranty introduces "this work.. I provide" and the final line of the conditions sums up the rhymes and what to do with (not shove) them.